src/resolvers/coordinates/query.ts

```diff
@@ -0,0 +1,5 @@
+export class CoordinatesQuery {
+    public async getCoordinates(parent: any, args: any, {dataSources}: any, info: any){
```

Comment:
why did you want to use `any` instead of actual types? Wouldn't using types better since we're using TypeScript?

Reply:
The only reason I didn't specify types there is that this  resolver just wire down the request to location service and do nothing else significant but yeah in context of typescript usage specifying types would make more sense.